lms/templates/student_profile/learner_profile.html

```diff
@@ -39,7 +41,8 @@
           (function (require) {
             require(['js/student_profile/views/learner_profile_factory'], function(setupLearnerProfile) {
                 var options = ${ json.dumps(data) };
-                setupLearnerProfile(options);
+                var platformName = ${ json.dumps(microsite.get_value('platform_name', settings.PLATFORM_NAME))};
```

Comment:
I was expecting this to be passed down as just another option, since that's how all the other settings come through. Why introduce another parameter just for this one?